partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -3375,634 +3927,633 @@ class PlanTyperTestsPorted {
-                name = "Duplicate fields in struct",
-                query = """
-                    SELECT a AS a
-                    FROM <<
-                        { 'a': 1, 'a': 'hello' }
-                    >> AS t
-                """,
-                expected = BagType(
-                    StructType(
-                        fields = listOf(
-                            StructType.Field("a", StaticType.unionOf(StaticType.INT4, StaticType.STRING))
-                        ),
-                        contentClosed = true,
-                        constraints = setOf(
-                            TupleConstraint.Open(false),
-                            TupleConstraint.UniqueAttrs(true),
-                            TupleConstraint.Ordered
-                        )
-                    )
-                )
-            ),
-            SuccessTestCase(
-                name = "Current User",
-                query = "CURRENT_USER",
-                expected = StaticType.unionOf(StaticType.STRING, StaticType.NULL)
-            ),
-            SuccessTestCase(
-                name = "Trim",
-                query = "trim(' ')",
-                expected = StaticType.STRING
-            ),
-            SuccessTestCase(
-                name = "Current User Concat",
-                query = "CURRENT_USER || 'hello'",
-                expected = StaticType.unionOf(StaticType.STRING, StaticType.NULL)
-            ),
-            SuccessTestCase(
-                name = "Current User Concat in WHERE",
-                query = "SELECT VALUE a FROM [ 0 ] AS a WHERE CURRENT_USER = 'hello'",
-                expected = BagType(StaticType.INT4)
-            ),
-            SuccessTestCase(
-                name = "TRIM_2",
-                query = "trim(' ' FROM ' Hello, World! ')",
-                expected = StaticType.STRING
-            ),
-            SuccessTestCase(
-                name = "TRIM_1",
-                query = "trim(' Hello, World! ')",
-                expected = StaticType.STRING
-            ),
-            SuccessTestCase(
-                name = "TRIM_3",
-                query = "trim(LEADING ' ' FROM ' Hello, World! ')",
-                expected = StaticType.STRING
-            ),
-            ErrorTestCase(
-                name = "TRIM_2_error",
-                query = "trim(2 FROM ' Hello, World! ')",
-                expected = StaticType.MISSING,
-                problemHandler = assertProblemExists {
-                    Problem(
-                        UNKNOWN_PROBLEM_LOCATION,
-                        PlanningProblemDetails.UnknownFunction(
-                            "trim_chars",
-                            args = listOf(StaticType.STRING, StaticType.INT4)
-                        )
+                        PlanningProblemDetails.UndefinedVariable(insensitive("pets"), emptySet())
                     )
                 }
             ),
+            // TestCase.ErrorTestCase(
```

Comment:
Did you mean to comment out all of these? I've gone through the post-merge commits, and there are only two things that immediately stand out to me. These commented out tests are one. The other has to do with the undefined variable returning ANY instead of MISSING.

Reply:
I did not, just need to isolate the failing tests. I will fix this ASAP